src/aio.rs

```diff
@@ -73,6 +90,27 @@ mod tokio_aio {
                 .await
                 .map(ActualConnection::TcpTokio)?)
         }
+        #[cfg(feature = "tls")]
+        async fn connect_tcp_tls(
+            hostname: &str,
+            socket_addr: SocketAddr,
+            insecure: bool,
+        ) -> RedisResult<ActualConnection> {
+            let tls_connector: tokio_tls::TlsConnector = if insecure {
+                TlsConnector::builder()
+                    .danger_accept_invalid_certs(true)
```

Comment:
That seems ... dangerous and I'd rather not set dangerous configuration by default.
Reducing the security of the TLS connection should be up to the user, not the library.

Reply:
Oh only now saw that this is behind the insecure flag. Ignore my comment then.

Reply:
Yeah, unfortunately it's a little hard to expose this `TlsConnector` to the user. We could figure it out, though.